auth/contexts/AuthenticatedUserContext.tsx

```
@@ -0,0 +1,122 @@
+          token,
+        });
+
+        setUser(response.user);
+        setAccessToken(response.accessToken);
+        setRefreshToken(response.refreshToken);
+        setExpiresAt(new Date().addSeconds(response.expiresIn));
+      },
+      async refreshToken(): Promise<void> {
+        await requestAndStoreFreshToken();
+      },
+      getToken: ((cb?: (token: string) => void): Promise<string> | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)